sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -364,6 +364,33 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
     }
   }
 
+  test("save csv with quote escaping enabled") {
+    withTempDir { dir =>
+      val csvDir = new File(dir, "csv").getCanonicalPath
+
+      val df = spark.createDataFrame(Seq(("test \"quote\"", 123,
+                                             "it \"works\"!", "\"very\" well")))
+        .toDF("a", "b", "c", "d")
```

Comment:
Indentation here.. maybe. I see other codes follow the indentations such as:

```scala
spark.createDataFrame(Seq(
  ("test \"quote\"", 123, "it \"works\"!", "\"very\" well")
)).toDF("a", "b", "c", "d")
```

```scala
val data = Seq(("test \"quote\"", 123, "it \"works\"!", "\"very\" well"))
spark.createDataFrame(data).toDF("a", "b", "c", "d")
```

and etc..